docs/scipp-neutron/instrument-view.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
     "# Instrument view\n",
     "\n",
-    "A simple version of the Mantid [instrument view](https://www.mantidproject.org/MantidPlot:_Instrument_View) is available in `scipp`. It currently does not support detector 'picking' and manual drawing of masks, but basic functionalities such as different 2D and 3D projections, as well as navigation through the time-of-flight dimension via a slider, are provided."
+    "A simple version of the Mantid [instrument view](https://www.mantidproject.org/MantidPlot:_Instrument_View) is available in `scipp`. It currently does not support detector 'picking' and manual drawing of masks, nor does it render the actualy shape of the detectors (currently it represents them as 2d squares), but basic functionalities spatial slicing, as well as navigation through the time-of-flight dimension via a slider, are provided."
```

Comment:
Yes, we used to support the bounding box and rotations, but this made the code quite messy. Having to deal with two different cases (full geometry and fast rendering), and well as having to guess the detector shape (cylinders or cuboids) depending on the name of the instrument. Plus having to manually create all the triangles for the mesh (because I never managed to get geometry instancing to work; see https://github.com/jupyter-widgets/pythreejs/issues/313). It was never optimal.

Maybe I should allow for different shapes (square, circle, rectangle) using texture mapping on the point cloud. But once again it will be sort of guessing, based on the instrument name.

I think until we actually read in directly the csg objects and get the triangles from the nexus geometry, we can stick with squares or circles. And once we are reading in the proper geometry, the point cloud should go away, so that we don't have to maintain the two different rendering options.